Gemfile

```diff
@@ -42,9 +42,9 @@ gem 'awrence' # convert snake_case hash keys to CamelCase hash keys
 gem 'bootstrap3-datetimepicker-rails'
 gem 'breadcrumbs_on_rails'
 gem 'builder'
-gem 'carmen' # countries and subdivisions
+gem 'carmen', '~>1.0.2'  # countries and subdivisions
 gem 'database_cleaner' # added to provide a solution to Capybara's problems with js=>true
-gem 'factory_girl_rails'
+gem 'factory_girl_rails', :require => true
```

Comment:
was this the gem who's location was disputed? Was that resolved?

Reply:
I don't think we have had the conversation John wanted to have about it yet.  It is resolved insomuch as it can be built and deployed correctly.

Reply:
This isn't perfect, but it works and Ryan is currently working another ticket to replace FactoryGirl with FactoryBot.  So, we are going with this.